.semaphore/semaphore.yml

```diff
@@ -71,7 +71,7 @@ blocks:
 
 - name: "API"
   run:
-    when: "change_in('/api/')"
+    when: "change_in(['/*', '/api/'], {exclude: ['/.semaphore', '/**/README.md', '/**/LICENSE']})"
```

Comment:
I'm not sure I understand `/**/README.md` - that matches every file named README.md in the root directory? Do we need the glob match or could we just do `/README.md`?

Reply:
I think that is every README in sub directories too no?

Reply:
Yes this is recursive .... so README in any directory under root
'/*' is non recursive - so that grabs only files directly in the root directory (Makefiles and go mod)